### PR TITLE
Update whats-new-windows-10-version-1809.md

### PR DESCRIPTION
**Pull request by RavennMSFT**
Added a note for Web sign-in

### CODE REVIEW

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
file diff:
```diff
- non-ADFS federated providers (e.g.SAML).
+ credentials not available on Windows (e.g. Temporary Access Pass). Going forward, Web sign-in will be restricted to only support Azure AD temporary access pass. 
```

File: windows/whats-new/whats-new-windows-10-version-1809.md [hunk 1]
```diff
@@ -224,7 +224,7 @@ Do you have shared devices deployed in your work place? **Fast sign-in** enables
 >[!IMPORTANT]
 >This is a private preview feature and therefore not meant or recommended for production purposes.
 
-Until now, Windows logon only supported the use of identities federated to ADFS or other providers that support the WS-Fed protocol. We are introducing “web sign-in,” a new way of signing into your Windows PC. Web Sign-in enables Windows logon support for non-ADFS federated providers (e.g.SAML).
+Until now, Windows logon only supported the use of identities federated to ADFS or other providers that support the WS-Fed protocol. We are introducing “web sign-in,” a new way of signing into your Windows PC. Web Sign-in enables Windows logon support for credentials not available on Windows (e.g. Temporary Access Pass). Going forward, Web sign-in will be restricted to only support Azure AD temporary access pass.
```
Comment (JohanFreelancer9) [inline windows/whats-new/whats-new-windows-10-version-1809.md:227]:
```suggestion
Until now, Windows logon only supported the use of identities federated to ADFS or other providers that support the WS-Fed protocol. We are introducing **web sign-in**, a new way of signing into your Windows PC. Web sign-in enables Windows logon support for credentials not available on Windows (for example, Azure AD temporary access pass). Going forward, web sign-in will be restricted to only support Azure AD temporary access pass. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - suggestions for consistency.

**Pull request merged**